addons/mail/static/src/core/web/@types/models.d.ts

```diff
@@ -1,15 +1,36 @@
 declare module "models" {
-    export interface Discuss  {
-        inbox: Thread,
-        stared: Thread,
-        history: Thread,
+    export interface Activity {
+        dateCreateFormatted: Readonly<string>;
+        dateDeadlineFormatted: Readonly<string>;
+        dateDoneFormatted: Readonly<string>;
+        edit: () => Promise<object>;
+        markAsDone: (attachmentIds: number[]) => Promise<void>;
+        markAsDoneAndScheduleNext: () => Promise<unknown>;
```

Comment:
I think this should return promise of `ActionDescription` from web action

Reply:
No way to resolve types coming from `orm.call(model, method)` but I'll add the JSDoc.